dts/arm/ti/j721e_main_r5.dtsi

```diff
@@ -55,6 +55,17 @@
 		status = "okay";
 	};
 
+	i2c6: i2c@2060000 {
```

Comment:
Could you add all the MAIN domain I2C ports while here? I know I2C6 is pinned out in an easy way on Beaglebone-AI64 and the one you tested with, but this is the SoC DTSI file and should be complete. Should be easy, all the addresses and IRQs are numbers sequential.

Reply:
No problem. I should've thought about it.
Will update it.